src/views/templates/details/tabs/details/components/TemplateDetailsLeftGrid.tsx

```diff
@@ -6,9 +6,10 @@ import DescriptionItem from 'src/views/templates/details/tabs/details/components
 import Labels from 'src/views/templates/details/tabs/details/components/Labels';
 import Name from 'src/views/templates/details/tabs/details/components/Name';
 import Namespace from 'src/views/templates/details/tabs/details/components/Namespace';
-import Owner from 'src/views/templates/details/tabs/details/components/Owner';
+// import Owner from 'src/views/templates/details/tabs/details/components/Owner';
```

Comment:
please remove

Reply:
Yeah, that was a bit of refactoring. The deleted `Owner` component was totally the same as we already have in utils.